content/md/products-super-power/work-on-your-assets.md

```diff
@@ -32,6 +33,29 @@ You can find a **locale and channel switcher** on each asset, meaning that if yo
 
 ![Assets Locale Switcher](../img/Assets_LocaleSwitcher.png)
 
+## Bulk edit a selection of asset attributes
+If you have the edit permission on an asset family, it is possible to bulk edit your own selection of asset attributes within this asset family.
+Let's imagine that you need to add the same tag to several assets, for instance their main color attribute. Instead of manually adding the tag for each one of these assets, you can easily do it in one action for your selection of assets, thanks to a bulk edition. Here's how to do so:
```

Comment:
```suggestion
Let's imagine that you need to add the same tag to several assets, for instance, their main color attribute. Instead of manually adding the tag for each of these assets, you can easily do it in one action for your selection of assets, thanks to a bulk edition. Here's how to do so:
```